docs/pages/app-signing/apple-developer-program-roles-and-permissions.mdx

```diff
@@ -7,7 +7,7 @@ description: Learn about the Apple Developer account membership requirements for
 import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
 import { Terminal } from '~/ui/components/Snippet';
 
-An Apple Developer account is required to create iOS device builds on EAS. This account allows you to generate [app signing credentials](/app-signing/managed-credentials/#generating-app-signing-credentials) such as certificates, identifiers, and profiles, submit the app for review, and manage app's distribution.
+An Apple Developer account with *App-Manager Role* is required to create iOS device builds on EAS. This account allows you to generate [app signing credentials](/app-signing/managed-credentials/#generating-app-signing-credentials) such as certificates, identifiers, and profiles, submit the app for review, and manage app's distribution.
```

Comment:
```suggestion
An Apple Developer account with at least App-Manager Role is required to create iOS device builds on EAS. This account allows you to generate [app signing credentials](/app-signing/managed-credentials/#generating-app-signing-credentials) such as certificates, identifiers, and profiles, submit the app for review, and manage app's distribution.
```

Since roles above in the hierarchy can also be used from an Apple account, I updated the above to provide that context.